www/src/js/reducers/settings.js

```diff
@@ -91,6 +93,12 @@ function settings(state: SettingsState = defaultSettingsState, action: FSA): Set
         moduleTableOrder: action.payload,
       };
 
+    case TOGGLE_BETA_TESTING_STATUS:
+      return {
+        ...state,
+        beta: !(state.beta || false),
```

Comment:
Not sure why you need the conditional, the default is already set.
```suggestion
        beta: !state.beta,
```

Reply:
Let me double check, but I think this was to make Flow happy

Reply:
Ah no, this works. Nice!